src/Chapter1.hs

```diff
@@ -416,7 +416,7 @@ task is to specify the type of this function.
 >>> squareSum 3 4
 49
 -}
-
+squareSum :: Num a => a -> a -> a
```

Comment:
We explain `Num` in the following chapters, so you are one step ahead 😸

In here `Int -> Int -> Int` would also work, as a more specific type 🙂

Reply:
Yeah, I think I like Int -> Int -> Int slightly better for my basic understanding. I'm still working through what the first was (the VS Code extension suggested it, so I cheated a bit there!).